lib/ansible/modules/packaging/os/snap.py

```diff
@@ -55,8 +55,8 @@
         default: stable
 
 author:
-    - Victor Carceler (vcarceler@iespuigcastellar.xeill.net)
-    - Stanislas Lange (angristan) <angristan@pm.me>
+    - Victor Carceler (@vcarceler) <vcarceler@iespuigcastellar.xeill.net>
+    - Stanislas Lange (@angristan) <<angristan@pm.me>
```

Comment:
Is `<<` intentional?

Reply:
Nope, that is a typo :-)

Reply:
OK, shall I do a PR to correct it? :)

Reply:
I already fixed it ;-)

Reply:
It's handy being able to push on `devel`... Thanks 😄